NEWS.md

```diff
@@ -1,4 +1,5 @@
-# drawCell 0.99.0
+# drawCell (development version)
 
 
 * Submission to Bioconductor 3.14
+* Improve shiny app with `shiny.semantic`
```

Comment:
We need to add here that now the pictures are vector images (higher quality).
Besides we also allow multi colouring, a feature that I did not implement before.